src/config.rs

```diff
@@ -187,22 +185,42 @@ impl Default for TermvizConfig {
     }
 }
 
-pub fn get_config() -> Result<TermvizConfig, confy::ConfyError> {
+pub fn ask_store() -> bool {
```

Comment:
there are crates that do this, i decided to reinvent the wheel to avoid more dependencies. Same with handling command line args